ClientManagementService/ClientManagementService.Domain/Mappers/Database/PetMapper.cs

```diff
@@ -21,6 +22,9 @@ public static CorePet ToCorePet(DbPet dbPet)
             corePet.OtherInfo = dbPet.OtherInfo;
             corePet.Picture = dbPet.Picture;
 
+            corePet.Owner = new Client() { Id = dbPet.OwnerId, FirstName = dbPet.Owner.FirstName, LastName = dbPet.Owner.LastName };

```

Comment:
This is same issue as above and it chains up.

Without "Include", Owner and BreedInfo is null. So it won't be mapped to Pet "Core". If it was not mapped to Pet Core, then it can't be mapped to DTO.

Reply:
I suggest:

1. Inject IBreedRepo and IClientRepo to PetService
2.  in all the GET methods in PetService, after you call get pet from DB, call get by id tp get client and breed. 
3. Then map it from DB Entity to Core Entity.

This way, no need to change DTO mapper because we guarantee that Core entity will have it from this mapping